src/main/java/mate/academy/dao/GenericDao.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.dao;
```

Comment:
I understand this. But also seems we need to make exactly like this:

 public interface MovieDao {
      Movie add(Movie movie);
  
      Optional<Movie> get(Long id);
  }